system/boot/armv7l/oemboot/suse-repart

```diff
@@ -934,7 +934,16 @@ function getFreeDiskBytes {
     if [ ! -b "$disk" ];then
         echo 0 ; return
     fi
+    local pt_type=$(blkid -s PTTYPE -o value $disk)
     local disk_Bytes=$(blockdev --getsize64 $disk)
+    # max msdos table geometry is at 2TB - 512
+    # max sector count taken from parted
+    local max_dos_disk=$((4294967295 * 512))
+    if [ "$pt_table_type" = "dos" ] && [ "$disk_Bytes" -gt $max_dos_disk ];then
```

Comment:
Where does pt_table_type get initialized? Is this transferred from some other script?

On line 937 pt_type is being set.